src/xcode/ENA/ENA/Source/Scenes/ENSetting/ExposureNotificationSettingViewController.swift

```diff
@@ -35,16 +35,16 @@ final class ExposureNotificationSettingViewController: UITableViewController {
 	private var lastActionCell: ActionCell?
 
 	let model = ENSettingModel(content: [.banner, .actionCell, .actionDetailCell, .descriptionCell])
-	let numberRiskContacts = 10
-	var enState: ENStateHandler.State
+	var state: State
 
 	init?(
 		coder: NSCoder,
 		initialEnState: ENStateHandler.State,
+		tracingHistory: TracingStatusHistory,
 		delegate: ExposureNotificationSettingViewControllerDelegate
 	) {
 		self.delegate = delegate
-		self.enState = initialEnState
+		self.state = State(enState: initialEnState, tracingHistory: tracingHistory)
```

Comment:
please give store instead of tracingHistory (later needed for additional PR) and I would prefer to not introduce a new State (It looks like React but it might be confusing later on what is ENState and what State)